api/server/server.go

```diff
@@ -30,14 +30,14 @@ const versionMatcher = "/v{version:[0-9.]+}"
 
 // Config provides the configuration for the API server
 type Config struct {
-	Logging          bool
-	EnableCors       bool
-	CorsHeaders      string
-	AuthZPluginNames []string
-	Version          string
-	SocketGroup      string
-	TLSConfig        *tls.Config
-	Addrs            []Addr
+	Logging                  bool
+	EnableCors               bool
+	CorsHeaders              string
+	AuthorizationPluginNames []string
```

Comment:
the configuration file uses the flag name, not the struct internal name, so it'd be `authorization-plugin`:

https://github.com/docker/docker/pull/18587#issuecomment-170665246